gk/main.c

```diff
@@ -33,8 +33,8 @@ gk_proc(void *arg)
 	uint32_t lcore = rte_lcore_id();
 	struct gk_config *gk_conf = (struct gk_config *)arg;
 
-	uint8_t port_in = get_net_conf()->front.id;
-	uint8_t port_out = get_net_conf()->back.id;
+	uint8_t port_in = get_if_front()->id;
+	uint8_t port_out = get_if_back()->id;
```

Comment:
I know that these two lines were changed as a reply to Cody's suggestion, but there's no need to use `get_if_front()` and `get_if_back()` in the C part of Gatekeeper since these functions just return fields that are hidden from LuaJIT. In order words, keep the original code.